Extractor/Zero2/Zero2Extractor.h

```diff
@@ -0,0 +1,39 @@
+
+#pragma once
+
+#include "../ZeroReader.h"
+
+class Zero2Reader : public ZeroReader
+{
+ private:
+  struct FileEntry
```

Comment:
Don't define the struct directly into the object. This struct gets redifined anyway in the Zero3 reader